examples/common/python/connectors/direct/work_order_jrpc_impl.py

```diff
@@ -91,22 +91,23 @@ def __validate_parameters(self, params, in_data, out_data):
                     logging.error("Missing in data parameter %s", k)
                     return False, "Missing in data parameter {}".format(k)
         
-        for data in out_data:
-            out_data_keys = []
-            for key in data.keys():
-                if not key in self.__data_key_map.keys():
-                    logging.error("Invalid out data parameter %s",key)
-                    return False, "Invalid out data parameter {}".format(key)
-                else:
-                    out_data_keys.append(key)
-            for k, v in self.__data_key_map.items():
-                if v == True and not k in out_data_keys:
-                    logging.error("Missing out data parameter %s", k)
-                    return False, "Missing out data parameter {}".format(k)
+        if out_data is not None:
+            for data in out_data:
```

Comment:
This code is duplicated for inData and outData validation. Could be moved to a function

Reply:
Fixed